### ISSUE TITLE
Use SANE Ember-cli version unless option is passed

### ISSUE THREAD

**Issue by IanVS** [artificialio/sane#138]
I think we could re-think the strategy of using SANE vs. global ember-cli (and sails).  Right now, there is a significant amount of code to simply determine what is installed and what the version numbers are.  This causes issues in Windows (see https://github.com/artificialio/sane/pull/129) and adds a lot of time to starting up a new SANE project.  

This could likely be streamlined by using `npm-which` and defaulting to a set behavior.  For example, by default SANE could use its own ember-cli unless a `--global-ember` variable is passed.  

I'd like to at least start a discussion around this, and find out what people think is the best way to handle ember-cli (and sails.js) versioning.  Do you like being given the option, even if it is at expense of `sane new` performance?  Would a default behavior (e.g. using the SANE versions) which also alerts you to the version numbers being used be sufficient?

**Comment by Globegitter** [comment]
Yeah you are right the current way slows creating a new project on Mac a few seconds and I believe more on Windows. A flag would certainly reduce that overhead because then we don't have to make any guesses and can immediately take the one or the other, I am just always careful with be flags because it is something to think about and one more thing for new developers. 

But I agree it would be good to streamline the behaviour and have a discussion what members of the community think.

**Comment by Globegitter** [comment]
I guess for other commands (at least on Linux / Mac) the questions becomes easier because we always now there is a sails and ember - cli already in the node_modules.

**Comment by krishnaglick** [comment]
I think it should default to the SANE CLI as the average user will probably be unaware of the option, and thus probably not need it. Plus the SANE CLI is, I assume, optimized for the SANE stack so it'd be the superior option in the first place.

**Comment by jamemackson** [comment]
One thing to keep in mind is that `sane new` is used once per project. If it were something I was running constantly I might be worried about a few seconds worth of performance but since it is not, I don't mind the wait.  

And I prefer the choice of the version. I might have different projects going that might not all be updated to the latest versions or something and having the choice is best I believe.  I could also see adding a flag to make the choice ahead of time but not just defaulting to something one way or another.
